handbook/people-ops/benefits-and-perks.md

```diff
@@ -2,13 +2,17 @@
 
 We want our teammates to be happy, healthy, and productive.
 
+## 🏡 All remote
+
+We are an [all-remote team](../../company/remote/index.md) so you never waste time commuting again, have flexible work hours, and work with the best people worldwide. We give you a [budget to either work from a coworking space or setup your home office](spending-company-money.md).
+
 ## 🏖️ Vacation
 
 We have a flexible vacation policy with mandatory annual vacation time that encourages our team to recharge when they need to. Read more about our philosophy on [mandatory vacation time](https://about.sourcegraph.com/blog/why-vacation-at-tech-companies-should-be-mandatory-better-code-happier-people).
 
 ## 🏥 Medical
 
-We offer high-quality medical and dental care for all full-time employees and their dependents.
+We offer high-quality medical and dental care for all full-time US employees and their dependents. If you a full-time teammate located outside of the US, your salary will be increased instead.
```

Comment:
```suggestion
We offer high-quality medical and dental care for all full-time US employees and their dependents. If you are a full-time teammate located outside of the US, your salary will be increased instead.
```

Reply:
How does this work in practice?

Reply:
This part of the diff is not a policy change; it only documents what we have already been doing. In the past we have given non-US teammates the option to structure their work as a contractor or an employee. When we present an offer the salary is based on a contractor rate, and so far everyone has chosen that option. If anyone had opted for an employment relationship, then we would have reduced the salary accordingly. When I review compensation on the engineering team, I adjust US teammates effective cash comp by adding an approximate cash value for these benefits.